src/components/commons/modal/BtnModal.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable no-unused-vars */
+import styled from '@emotion/styled';
+import React, { useRef, useState } from 'react';
+import { CloseIc } from '../../../assets/svgs';
+import useClickOutside from '../../../hooks/useClickOutside';
+
+interface BtnCloseModalPropType {
+  title: string;
+  showModal: boolean;
+  handleShowModal: (type: boolean) => void;
```

Comment:
```suggestion
  isModalOpen: boolean;
  handleModalOpen: () => void;
```

p3) 전에도 남겼던 코리인데, boolean prop 네이밍은 요런식으로 Is~ 이렇게 하는거 어때요?? 
비슷하게 handler 이름도 바꿔봤어요!